cmd/provisioner-localpv/app/helper_hostpath.go

```diff
@@ -213,3 +219,24 @@ func (p *Provisioner) exitPod(hPod *corev1.Pod) error {
 	}
 	return nil
 }
+
+// getToleration() returns the tolerations, for the respective taints.
```

Comment:
Move this code that converts taints to tolerations to the pod builder new function as suggested below.